tool_collections/taxonomy/kraken2tax/kraken2tax.xml

```diff
@@ -0,0 +1,102 @@
+<tool id="Kraken2Tax" name="Convert Kraken" version="1.0">
+  <description>data to Galaxy taxonomy representation</description>
+    <requirements>
+        <requirement type="package" version="8d245994d7">gb_taxonomy</requirement>
+    </requirements>
+  <command>
+<![CDATA[
+
+  awk '{ print \$${read_name}, \$${tax_id} }' OFS="\t" "${input}" | taxonomy-reader \${GALAXY_DATA_INDEX_DIR}/taxonomy/names.dmp \${GALAXY_DATA_INDEX_DIR}/taxonomy/nodes.dmp 1 > "${out_file}"
+
+  ## To function this tool requires names.dmp and nodes.dmp files dowloaded from ftp://ftp.ncbi.nih.gov/pub/taxonomy/taxdump.tar.gz
+  ## These two files must be placed into a filesystem location accessible at ${GALAXY_DATA_INDEX_DIR}/taxonomy/
```

Comment:
Not sure I like this. Isn't the proper way to either write a data manager for it or use a `.loc` file (maybe even with different versions)?